src/components/svgs/lines/styles/bjsubway-dotted.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { CityCode, MonoColour } from '@railmapgen/rmg-palette-resources';
+import {
+    LinePathAttributes,
+    LinePathType,
+    LineStyle,
+    LineStyleComponentProps,
+    LineStyleType,
+} from '../../../../constants/lines';
+import { ColorField, AttributesWithColor } from '../../../panels/details/color-field';
+
+const BjsubwayDotted = (props: LineStyleComponentProps<BjsubwayDottedAttributes>) => {
+    const { id, path, styleAttrs, handleClick } = props;
+    const { color = defaultBjsubwayDottedAttributes.color } = styleAttrs ?? defaultBjsubwayDottedAttributes;
+
+    const onClick = React.useCallback(
+        (e: React.MouseEvent<SVGPathElement, MouseEvent>) => handleClick(id, e),
+        [id, handleClick]
+    );
+
+    return (
+        <g id={id}>
+            <path d={path} fill="none" stroke={color[2]} strokeWidth="5" strokeDasharray="2 2" />
+            <path d={path} fill="none" stroke="white" strokeWidth="3.8" />
```

Comment:
It would be great to have this automatically switched by drak mode :)
https://github.com/railmapgen/rmp/blob/904d178402a720d091480f306784962803fcb944/src/components/svgs/stations/gzmtr-int.tsx#L57

Reply:
Fixed in [4364f82](https://github.com/railmapgen/rmp/pull/430/commits/4364f8238314e347d382ec0e5b8c678a7d60e3a2)